assets/app/views/directives/logs/_log-viewer.html

```diff
@@ -3,6 +3,18 @@
     <span>Current log from {{start}}&nbsp;</span>
     <span ng-if="end">to {{end}}</span>
   </div>
+
+  <div row main-axis="end" ng-if="canViewArchive">
+    <form
+      ng-if="canViewArchive"
+      action="{{kibanaAuthUrl}}"
+      method="POST" target="_blank">
```

Comment:
Yeah and I dont want us touching anything to do with the java console
integration this late today.  So make kibana links consistent with that.

On Wed, Nov 4, 2015 at 9:57 AM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/directives/logs/_log-viewer.html
> <https://github.com/openshift/origin/pull/5663#discussion_r43889622>:
>
> > @@ -3,6 +3,18 @@
> >      <span>Current log from {{start}}&nbsp;</span>
> >      <span ng-if="end">to {{end}}</span>
> >    </div>
> > +
> > +  <div row main-axis="end" ng-if="canViewArchive">
> > +    <form
> > +      ng-if="canViewArchive"
> > +      action="{{kibanaAuthUrl}}"
> > +      method="POST" target="_blank">
>
> The idea is that Kibana and the Java console should look like the web
> console so they feel like one integrated application. I'm not sure we've
> accomplished that.
>
> I think we should be consistent in how we handle the links to both
> whichever way we go.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5663/files#r43889622>.
>